test/Dapr.Actors.Test/ActorProxyTests.cs

```diff
@@ -0,0 +1,41 @@
+// ------------------------------------------------------------
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT License.
+// ------------------------------------------------------------
+
+namespace Dapr.Actors.Client
+{
+    using System;
+    using Dapr.Actors.Builder;
+    using Dapr.Actors.Client;
+    using Dapr.Actors.Test;
+    using FluentAssertions;
+    using Xunit;
+
+    /// <summary>
+    /// Test class for Actor Code builder.
+    /// </summary>
+    public class ActorProxyTests
+    {
+        /// <summary>
+        /// Tests Proxy Creation.
+        /// </summary>
+        [Fact]
+        public void TestCreateActorProxy_Success()
+        {
+            var actorId = new ActorId("abc");
+            var proxy1 = ActorProxy.Create(actorId, "TestActor");
+            Assert.NotNull(proxy1);
```

Comment:
can you please split these up? these are two different tests for two different APIs